sync-version/action.yaml

```diff
@@ -21,6 +24,34 @@ runs:
       with:
         token: ${{ inputs.token }}
 
+    - name: Find the snapcraft.yaml path
+      id: yaml-path
+      shell: bash
+      run: |
+        if [[ -n "${{ inputs.snapcraft-yaml-path }}"; then
+          yaml-path="${{ inputs.snapcraft-yaml-path }}"
```

Comment:
Per my other comment - you can't use `-` as a separator in bash variables

I'd suggest the following convention across both PRs

- Github Inputs and Outputs are lower case, separated by `-`
- Bash variable names are lower case, separated by `_`

Reply:
Check now, if they are okay